hubblestack_nebula/hubblestack_nebula_queries.yaml

```diff
@@ -24,7 +24,7 @@ day:
   - query_name: uptime
     query: SELECT t.unix_time AS query_time, total_seconds AS uptime FROM uptime LEFT JOIN time as t;
   - query_name: suid_binaries
-    query: SELECT t.unix_time AS query_time, sb.*, f.uid, f.gid, f.mode AS file_acl, h.md5, h.sha1, h.sha256 FROM suid_bin AS sb JOIN time AS t LEFT JOIN hash AS h ON sb.path=h.path LEFT JOIN file AS f on sb.path=f.path;
+    query: SELECT t.unix_time AS query_time, sb.username AS user, sb.groupname AS 'group', sb.permissions, sb.path AS object_path, f.uid, f.gid, f.mode AS file_acl, h.md5, h.sha1, h.sha256 FROM suid_bin AS sb JOIN time AS t LEFT JOIN hash AS h ON sb.path=h.path LEFT JOIN file AS f on sb.path=f.path;
```

Comment:
_group_ is a sql keyword, so as soon as you have standalone word called _group_, sqlite part of osquery throws an error. I've found no other way to get past this issue.